README.md

```diff
@@ -158,13 +158,25 @@ You can also test it on the command line, for example:
 
 ### Starting the Web App
 
-#### 1. Installing dependencies
+#### 1. Check out the code
+
+Clone the Image Caption Generator Web App repository locally then run the following command:
+
```

Comment:
Shouldn't this be "Clone the Image ... by running the following command"?

Reply:
nice catch, I was probably thinking of the next step while typing.